metadata/reportssql/hivtmi_report_functions.sql

```diff
@@ -101,59 +75,9 @@ SELECT
 FROM
   patient pat
 WHERE
-  getObsDatetimeValueInSection(pat.patient_id, "c6c08cdc-18dc-4f42-809c-959621bc9a6c", "b70dfca0-db21-4533-8c08-4626ff0de265") BETWEEN p_startDate AND p_endDate AND
+  getObsDatetimeValueInSection(pat.patient_id, "c6c08cdc-18dc-4f42-809c-959621bc9a6c", "d6cc3709-ffa0-42eb-b388-d7def4df30cf") BETWEEN p_startDate AND p_endDate AND
```

Comment:
@tresorluz why did you change the parent concept (i.e. the second uuid parameter) here? The new concept uuid used refers to "At ANC Enrolment" which is not the right parent concept for HIV Test Date. The old uuid value (i.e. "b70dfca0-db21-4533-8c08-4626ff0de265") was the right one, it refers to the sub-section "HIV Test".

Reply:
We made this change with you because this concept b70dfca0-db21-4533-8c08-4626ff0de265 is in hiv testing and counseling form not in ANC initial form